config/default.yml

```
@@ -82,6 +82,11 @@ Minitest/AssertRaisesCompoundBody:
   Enabled: pending
   VersionAdded: '0.21'
 
+Minitest/AssertRaisesWithRegexpArgument:
+  Description: 'This cop enforces checks for regular expression literals passed to `assert_raises`.'
+  Enabled: pending
+  VersionAdded: '0.22'
```

what should these values be?

---

Can you update to the below?

```suggestion
  VersionAdded: '<<next>>'
```